### ISSUE TITLE
Bash completion for available models and commands needs to be speeded up

### ISSUE THREAD

**Issue by simonzhaoms** [mlhubber/mlhub#34]
Completion is slow:
```console
$ ml install [TAB] [TAB]
barchart             colorize             iris                 object-recognition   rain                 
clothes-recommender  dogs-cats            movie-recommender    ports-plots          scatter-plot
$ ml d[TAB] [TAB]
demo     display
```

**Comment by simonzhaoms** [comment]
Fixed in commit 05a961a

**Comment by simonzhaoms** [comment]
And enhanced in new commit https://github.com/mlhubber/mlhub/commit/43a4745bbd16cc0738c34055ff0775e9096d2dba
Give it a try! 😃
